docs/src/main/sphinx/installation/cli.rst

```diff
@@ -77,9 +77,37 @@ The detailed behavior is as follows:
   the query.
 * Further queries in the CLI session do not require additional logins while the
   authentication token remains valid. Token expiration depends on the external
-  authentication system configuration.
+  authentication type configuration.
 * Expired tokens force you to log in again.
 
+.. _cli-certificate-auth:
+
+Certificate authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the following CLI command line arguments to connect to a cluster that uses
+:doc:`certificate authentication </security/certificate>`.
+
+.. list-table:: CLI options for ertificate authentication
+   :widths: 35 65
+   :header-rows: 1
+
+   * - Option
+     - Description
+   * - ``--keystore-path=<path>``
+     - Absolute or relative path to a :doc:`PEM </security/inspect-pem>` or
+       :doc:`JKS </security/inspect-jks>` file, which must contain a certificate
+       that is trusted by the Trino cluster you are connecting to.
+   * - ``--keystore-password=<password>``
+     - Only required if the keystore has a password.
+   * - ``--keystore-type=<type>``
+     - Rarely used. Specifies the value of the keystore.type property, or ``jks``
+       if not specified. See the ``java.security`` file in your JDK
+       installation's ``lib/security`` or ``conf/security`` directory.
+
+The three ``--truststore`` related options are not used for client certificate
```

Comment:
Maybe clarify that 

> These options control the client's trust of the server's certificate.

Since it sounds like you shouldn't use them at all. Or perhaps we should say "are independent of" rather than "not used"

Reply:
went with both suggestions